### DISCUSSION TITLE
No info about disk used and total space

### DISCUSSION THREAD

**Discussion by TornaxO7** [nicolargo/glances#2855] [Q&A]
![image](https://github.com/nicolargo/glances/assets/50843046/5162d7c2-605f-4373-a93b-745b8ab5094d)

Hi. I'm not able to get any information about the disk usage and total size. Does anyone has got an idea how to fix this?
I've installed `glance` with docker through NixOS:
```nix
    virtualisation.oci-containers.containers.glances = {
      image = "docker.io/nicolargo/glances:latest-full";
      volumes = [
        "/run/user/1000/podman/podman.sock:/run/user/1000/podman/podman.sock:ro"
      ];
      ports = [
        "61208:61208"
      ];
      environment = {
        "GLANCES_OPT" = "-w";
        "TZ" = "Europe/Berlin";
      };
      extraOptions = [
        "--network=host"
        "--privileged"
      ];
    };
```
Did I miss something?

**Comment by nicolargo** [comment]
Hi @TornaxO7 

i did not know NixOS but perhaps it did not expose host disk inside containers.

Try to add the disk you want to monitor in the **volumes** list.
